modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -99,6 +106,25 @@ const isEmptyTextNode = (node: Node | null): boolean => {
   return Type.isNonNullable(node) && NodeType.isText(node) && node.length === 0;
 };
 
+const isWrapNoneditableTarget = (editor: Editor, node: Node): boolean => {
+  const selectors = [ '[data-mce-cef-wrappable]' ].concat(Options.getWrapNoneditableSelectors(editor)).join(',');
+  return Selectors.is(SugarElement.fromDom(node), selectors);
+};
+
```

Comment:
This is where a non-editable element is determined to be wrappable.

At the moment, either a selector provided by the new option or `data-mce-cef-wrappable` attribute can be used to indicate it is wrappable. It was discussed during the knowledge share that having both could be useful so that we can use the data attribute in our plugins/code without worrying about what the integrator sets the option to.
Are we happy with `data-mce-cef-wrappable` as the attribute name?

In addition, as noted in the comment, I have made it so that a non-editable element can only be wrapped if it has no editable descendants. If there are concerns about that, a follow-up task can be created.